docs/src/components/navigation/nav-item.tsx

```diff
@@ -22,23 +30,42 @@ export function NavItem({ label, href, level, title, tabIndex }: NavItemProps) {
   }`;
 
   const styleIdle =
-    'hover:bg-keystatic-gray-light active:bg-keystatic-gray font-medium text-neutral-600';
+    'font-medium text-neutral-600 hover:bg-keystatic-gray-light active:bg-keystatic-gray';
 
   const styleCurrent =
     'bg-keystatic-gray text-keystatic-gray-dark font-semibold before:block before:absolute before:inset-y-1 before:-inset-x-2 before:bg-keystatic-gray-dark before:w-1 before:rounded before:transition-all';
 
   return (
     <li className="relative" aria-current={isCurrentPage ? 'page' : false}>
-      <Link href={href} legacyBehavior>
-        <a
-          className={cx(styleShared, isCurrentPage ? styleCurrent : styleIdle)}
-          href={href}
-          title={title}
-          tabIndex={tabIndex}
+      {comingSoon ? (
+        <div
+          className={cx(
+            styleShared,
+            'inline-flex gap-1 items-baseline text-neutral-500'
+          )}
         >
           {label}
-        </a>
-      </Link>
+          {comingSoon && (
```

Comment:
You don't need this check since you're doing it on line 40. Sorry, I missed this earlier.

Reply:
Ahh thank you 😅🙈